### DISCUSSION TITLE
Can this plugin work with MEGA?

### DISCUSSION THREAD

**Discussion by David7ce** [remotely-save/remotely-save#859] [Ideas]
For me MEGA is the best free cloud storage, better than Dropbox and GDrive, has 20 GiB and the client is a real cross-platform app (multiple Linux distros, macOS, Windows, iOS, Android and web-app).
I'd love to see an integration with MEGA.

Thanks, great Obsidian plugin!

**Comment by fyears** [comment]
i would like to, and i have done some researches. but mega doesn't have official or good 3rd js library so it's technically hard to supoort mega

**Reply by David7ce** [reply]
> i would like to, and i have done some researches. but mega doesn't have official or good 3rd js library so it's technically hard to supoort mega

Oh that's a bit sad. In any case I found a solution to my workflow where paying MEGA premium I can synchronize my Android phone with the cloud, and in that way synchronize easily notes from my computers with phone and edit them with Obsidian.

This [new synchronization feature of MEGA](https://help.mega.io/installs-apps/mobile/set-up-android-sync) is only available for Android users, previously you have to use a 3rd party tool to sync the phone with the cloud.

**Comment by shlok-ps** [comment]
Mega cli has webdav feature. Run the webdav on a server, integrate it with remotely-save
